hazelcast/src/main/java/com/hazelcast/jet/impl/JobRepository.java

```diff
@@ -519,46 +526,62 @@ private boolean isResourceMapExpired(long creationTime) {
 
     Set<Long> getAllJobIds() {
         Set<Long> ids = new HashSet<>();
-        ids.addAll(jobRecords.keySet());
-        ids.addAll(jobResults.keySet());
+        ids.addAll(jobRecords.get().keySet());
```

Comment:
I think we should use the `jobRecordsMap` and `jobResultsMap` here as well.

Reply:
👍